packages/sfc-playground/src/App.vue

```diff
@@ -1,13 +1,8 @@
 <script setup lang="ts">
 import Header from './Header.vue'
-import { Repl, ReplStore } from '@vue/repl'
+import { Repl, ReplStore, SFCOptions } from '@vue/repl'
 import { ref, watchEffect } from 'vue'
-
-const setVH = () => {
-  document.documentElement.style.setProperty('--vh', window.innerHeight + `px`)
```

Comment:
This is actually intentional because `100vh` on mobile includes browser UI height, which makes it bigger than `window.innerHeight` (size of the viewport users can actually see)

Reply:
The new, dynamic viewport units can be used, then?
https://web.dev/viewport-units/
https://caniuse.com/viewport-unit-variants